README.md

```diff
@@ -1 +1,84 @@
-# To-Dos-App
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> "ToDoApp" is a tool that helps to organize your day. It creates a list of chores that you need to do and allows you to mark them as complete, edit and delete them.
+
+
+## Built With
+
+-JavaScript
+- ReactJs
+- Git, GitHub, Git workflow
+
+## Live Demo (if available)
```

Comment:
Please, I suggest you remove the Live Demo if there's no link yet, or remove the statement (if available) 😀